domains/client/domain-operator/src/tests.rs

```diff
@@ -3547,385 +3546,6 @@ async fn test_cross_domains_messages_should_work() {
-//     })
-//     .await
-//     .unwrap();
-//
-//     // Open channel between the core payments domain and the core eth relay domain
-//     let fee_model = FeeModel {
-//         outbox_fee: ExecutionFee {
-//             relayer_pool_fee: 1,
-//             compute_fee: 5,
-//         },
-//         inbox_fee: ExecutionFee {
-//             relayer_pool_fee: 2,
-//             compute_fee: 3,
-//         },
-//     };
-//     charlie
-//         .construct_and_send_extrinsic(pallet_sudo::Call::sudo {
-//             call: Box::new(core_eth_relay_domain_test_runtime::RuntimeCall::Messenger(
-//                 pallet_messenger::Call::initiate_channel {
-//                     dst_domain_id: DomainId::CORE_PAYMENTS,
-//                     params: InitiateChannelParams {
-//                         max_outgoing_messages: 100,
-//                         fee_model,
-//                     },
-//                 },
-//             )),
-//         })
-//         .await
-//         .expect("Failed to construct and send extrinsic");
-//     // Wait until channel open
-//     produce_blocks_until!(ferdie, alice, bob, charlie, {
-//         bob.get_open_channel_for_domain(DomainId::CORE_ETH_RELAY)
-//             .is_some()
-//             && charlie
-//                 .get_open_channel_for_domain(DomainId::CORE_PAYMENTS)
-//                 .is_some()
-//     })
-//     .await
-//     .unwrap();
-//
-//     // Transfer balance cross the core payments domain and the core eth relay domain
-//     let pre_bob_free_balance = bob.free_balance(bob.key.to_account_id());
-//     let pre_charlie_free_balance = charlie.free_balance(charlie.key.to_account_id());
-//     let transfer_amount = 10;
-//     bob.construct_and_send_extrinsic(pallet_transporter::Call::transfer {
-//         dst_location: pallet_transporter::Location {
-//             domain_id: DomainId::CORE_ETH_RELAY,
-//             account_id: AccountIdConverter::convert(Charlie.into()),
-//         },
-//         amount: transfer_amount,
-//     })
-//     .await
-//     .expect("Failed to construct and send extrinsic");
-//     // Wait until transfer succeed
-//     produce_blocks_until!(ferdie, alice, bob, charlie, {
-//         let post_bob_free_balance = bob.free_balance(bob.key.to_account_id());
-//         let post_charlie_free_balance = charlie.free_balance(charlie.key.to_account_id());
-//
-//         post_bob_free_balance
-//             == pre_bob_free_balance
-//                 - transfer_amount
-//                 - fee_model.outbox_fee().unwrap()
-//                 - fee_model.inbox_fee().unwrap()
-//             && post_charlie_free_balance == pre_charlie_free_balance + transfer_amount
-//     })
-//     .await
-//     .unwrap();
-// }
-
-// TODO: Unlock test when multiple domains are supported in DecEx v2.
-// #[tokio::test(flavor = "multi_thread")]
-// async fn test_unordered_cross_domains_message_should_work() {
```

Comment:
This test is worth keeping IMO, plz leave it commented out here I will revisit it as I'm the original author.